sdk/metricsadvisor/Azure.AI.MetricsAdvisor/perf/Azure.AI.MetricsAdvisor.Perf/Infrastructure/MetricsAdvisorTest.cs

```diff
@@ -1,25 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using Azure.Test.Perf;
-
-namespace Azure.AI.MetricsAdvisor.Perf
-{
-    public abstract class MetricsAdvisorTest<TOptions> : PerfTest<TOptions> where TOptions : PerfOptions
```

Comment:
Curious: why are these getting deleted?  Is it temporary while we do the exploration?

Reply:
Yes, I'll add back the live tests to prove the grow up story.